app/_gateway_entities/key.md

```diff
@@ -20,16 +18,28 @@ api_specs:
 tools:
   - admin-api
   - konnect-api
-  - kic
-  - deck
-  - terraform
 
 schema:
   api: gateway/admin-ee
   path: /schemas/Key
-
 ---
 
-@todo
+## What is a Key?
+
+A Key object holds a representation of asymmetric keys, either public or private. When Kong or a Kong plugin requires a specific public or private key to perform certain operations, it can use this entity.
```

Comment:
```suggestion
A Key object holds a representation of asymmetric keys, either public or private. When {{site.base_gateway}} or a Kong plugin requires a specific public or private key to perform certain operations, it can use this entity.
```